code/tier_classification/tier_classification.R

```diff
@@ -0,0 +1,247 @@
+# Author: Run Jin
+# Classify SNV mutation results into 4 tier [@doi:10.1016/j.jmoldx.2016.10.002]
+
+suppressPackageStartupMessages({
+  library(optparse)
+  library(tidyverse)
+  library(readr)
+})
+
+# function to get all directories
+
+# Define Directories
+root_dir <- rprojroot::find_root(rprojroot::has_dir(".git"))
+data_dir <- file.path(root_dir, "data")
+input_dir <- file.path(patient_dir, "output", "oncokb_analysis")
+
+# Read in files necessary for analyses
+oncokb_anno <- readr::read_tsv(file.path(input_dir, "oncokb_consensus_annotated.txt"))
+
+# Read in cancer genes list from OMPARE knowledge base
+cancer_genes <- readRDS(file.path(data_dir, 'cancer_gene_list.rds'))
+
+# Read in hotspot database
+hotspot_indel <- readr::read_tsv(file.path(data_dir, 'hotspots_database', 'hotspot_database_2017_indel.tsv')) %>% 
+  distinct()
+hotspot_snv <- readr::read_tsv(file.path(data_dir, 'hotspots_database', 'hotspot_database_2017_snv.tsv')) %>%
+  distinct()
+
```

Comment:
Maybe we can filter the hotspots to genes corresponding to VUS/Mutations from all_findings_output (which is a superset of key_clinical_findings_output) so that the loop only runs on genes of interest. 

```suggestion
# subset to genes in all_findings_output corresponding to VUS and Mutation
genes_of_interest <- all_findings_output %>%
  filter(Type %in% c("VUS", "Mutation")) %>% 
  .$Aberration
hotspot_snv <- hotspot_snv %>% 
  filter(Hugo_Symbol %in% genes_of_interest)
hotspot_indel <- hotspot_indel %>%
  filter(Hugo_Symbol %in% hotspot_snv)
```